backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/impl/BPartnerStatsDAO.java

```diff
@@ -110,16 +110,14 @@ public BigDecimal retrieveOpenItems(@NonNull final BPartnerStats bpStats)
 
 		BigDecimal openItems = null;
 
-		final Object[] sqlParams = new Object[] { stats.getC_BPartner_ID() };
 		final String sql = "SELECT "
-				+ "currencyBase(openamt,C_Currency_ID,DateInvoiced,AD_Client_ID,AD_Org_ID) from de_metas_endcustomer_fresh_reports.OpenItems_Report(now()::date) where  C_BPartner_ID=?";
+				+ "COALESCE((SELECT SUM(currencyBase(openamt,C_Currency_ID,DateInvoiced,AD_Client_ID,AD_Org_ID)) from de_metas_endcustomer_fresh_reports.OpenItems_Report(now()::date, " + stats.getC_BPartner_ID() + ")), 0)";
```

Comment:
pls use `?` instead of concatenating the value

Reply:
(also thx for fixing the `SUM`-thingie :-) )